alphagov/paas-cf
# [#126491293] Add Ben Hyland's GPG key to admin users and merge keys.

## What

This adds his key to the list of admin users (so staging/prod accounts get created), and to the list of keys allowed to sign commits on master (so that he can merge PRs).

Along with this, I've also tidied up the list of admin users (alphabetised it), and added a make target to list details of the merge keys.

## How to review

* Verify that the added key does indeed belong to Ben Hyland.
* Verify that the tidying up is purely a refactoring, and hasn't changed any user details.
* Verify that the new make target works.

## Who can review

Anyone with existing merge access except for myself.
